applications/polyp_detection/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# syntax=docker/dockerfile:1
+
+# SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+FROM nvcr.io/nvidia/clara-holoscan/holoscan:v2.7.0-dgpu
```

Comment:
Hi @jjomier @MostafaToloui , currently I used this base image, I'm not sure if we need to build based on a publicly available image, do you have any suggestions? Thanks!

Reply:
Why don't we use the latest Holoscan image. It is passed by default when building the container (you can look at other applications).